bench/simple_consumer_bench.py

```diff
@@ -0,0 +1,111 @@
+import json
```

Comment:
Before merging this file, it would be good for it to gain some docstrings and usage examples. It looks really useful, but I have to pick through the code to understand how it works.